src/hijacker.js

```diff
@@ -79,12 +79,13 @@
 
             responseHeadersString = xhr.getAllResponseHeaders();
 
-            if (!this.responseHeaders) {
-                this.responseHeaders = {};
-                while ((match = rHeaders.exec(responseHeadersString))) {
-                    this.responseHeaders[match[1].toLowerCase()] = match[2];
-                }
+            // We need to clear the responseHeaders object every time, otherwise
+            // we might get a content type mismatch which causes 🔥💣🔥
+            this.responseHeaders = {};
```

Comment:
@henryh15 as discussed in meat space, just make `responseHeaders` a local variable in this function. Since we're using a singleton Hijacker object, assigning this to the instance doesn't make sense.